src/vivarium_gates_nutrition_optimization/data/loader.py

```diff
@@ -289,6 +296,79 @@ def load_pregnant_maternal_hemorrhage_incidence(key: str, location: str):
     return maternal_hemorrhage_incidence.applymap(lambda value: 1 if value > 1 else value)
 
 
+def load_hemoglobin_maternal_hemorrhage_rr(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.MATERNAL_HEMORRHAGE.RR_ATTRIBUTABLE_TO_HEMOGLOBIN:
+        raise ValueError(f"Unrecognized key {key}")
+
+    distribution = data_values.RR_MATERNAL_HEMORRHAGE_ATTRIBUTABLE_TO_HEMOGLOBIN
+    dist = sampling.get_lognorm_from_quantiles(*distribution)
+    # Get a DataFrame with the desired index
+    demographic_dimensions = get_data(data_keys.POPULATION.DEMOGRAPHY, location)
+
+    rng = np.random.default_rng(get_hash(f"{key}_{location}"))
+    maternal_hemorrhage_rr = pd.DataFrame(
+        np.tile(dist.rvs(size=1000, random_state=rng), (len(demographic_dimensions), 1)),
+        columns=vi_globals.DRAW_COLUMNS,
+        index=demographic_dimensions.index,
+    )
+    return maternal_hemorrhage_rr
+
+
+def load_hemoglobin_maternal_hemorrhage_paf(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.MATERNAL_HEMORRHAGE.PAF_ATTRIBUTABLE_TO_HEMOGLOBIN:
+        raise ValueError(f"Unrecognized key {key}")
+
+    rr = get_data(data_keys.MATERNAL_HEMORRHAGE.RR_ATTRIBUTABLE_TO_HEMOGLOBIN, location)
+    proportion = get_data(
+        data_keys.HEMOGLOBIN.PREGNANT_PROPORTION_WITH_HEMOGLOBIN_BELOW_70, location
+    )
+
+    return (rr * proportion + (1 - proportion) - 1) / (rr * proportion + (1 - proportion))
+
+
+def load_hemoglobin_maternal_disorders_rr(key: str, location: str) -> pd.DataFrame:
+    if key != data_keys.MATERNAL_DISORDERS.RR_ATTRIBUTABLE_TO_HEMOGLOBIN:
+        raise ValueError(f"Unrecognized key {key}")
+
+    groupby_cols = ["age_group_id", "sex_id", "year_id"]
+    draw_cols = [f"draw_{i}" for i in range(1000)]
+    rr = extra_gbd.get_hemoglobin_maternal_disorders_rr()
+    rr = rr.groupby(groupby_cols)[draw_cols].sum().reset_index()
+    rr = reshape_to_vivarium_format(rr, location)
+    return rr
+
+
+def load_hemoglobin_maternal_disorders_paf(key: str, location: str) -> pd.DataFrame:
+    location_id = utility_data.get_location_id(location)
+    demography = get_data(data_keys.POPULATION.DEMOGRAPHY, location)
+
+    data = pd.read_csv(paths.HEMOGLOBIN_MATERNAL_DISORDERS_PAF_CSV)
+    data = data.set_index("location_id").loc[location_id]
+    age_bins = utility_data.get_age_bins()
+    data = data.merge(age_bins, on="age_group_id")
+    data.draw = data.draw.apply(lambda d: f"draw_{d}")
+    data = data.pivot(index=["age_start", "age_end"], columns="draw", values="paf")
+    data = data.reset_index(level="age_end", drop=True).reindex(
+        demography.index, level="age_start", fill_value=0.0
+    )
+    return data
+
+
+def get_moderate_hemorrhage_probability(key: str, location: str) -> pd.DataFrame:
```

Comment:
this function is new with respect to IV iron (the hemorrhage severity is handled elsewhere)